manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1

```

```suggestion
PackageVersion: 4.2
```
to match Control Panel.